lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGNR(Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGNR(Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
```

Comment:
ah i see- so GLOBAL_FORCE_USE_WRITE_BARRIER does not actually force using software write barrier? it only allows you to use software write barrier everywhere? seems confusing- although given the explanation, I see the value of EnableHardwareWriteBarrier, but I would change some name (maybe GLOBAL_FORCE_USE_WRITE_BARRIER) so this isn't as confusing

---
In reply to: [87932027](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r87932027) [](ancestors = 87932027,87931407,87930847,87929333,87929125)